django/views/debug.py

```diff
@@ -112,6 +72,16 @@ class ExceptionReporterFilter:
     contain lenient default behaviors.
     """
 
+    DEFAULT_CLEANSED_SUBSTITUTE = '********************'
+    DEFAULT_HIDDEN_SETTINGS = re.compile('API|TOKEN|KEY|SECRET|PASS|SIGNATURE', flags=re.IGNORECASE)
+
+    def __init__(self, cleansed_substitute=DEFAULT_CLEANSED_SUBSTITUTE, hidden_settings=DEFAULT_HIDDEN_SETTINGS):
```

Comment:
Make these defaults `None`. (Then below set `self.CLEANSED_SUBSTITUTE` if `not None`.)